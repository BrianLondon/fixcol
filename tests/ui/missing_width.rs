use fixed_derive::{ReadFixed};

#[derive(ReadFixed)]
struct Point {
    #[fixed(width = 5)]
    x: u16,
    y: u16,
}

pub fn main() {}
