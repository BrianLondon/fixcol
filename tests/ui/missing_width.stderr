error: fixed-derive error: Width must be specified for all fields.
 --> tests/ui/missing_width.rs:7:5
  |
7 |     y: u16,
  |     ^
